branch-2.1: [chore](binlog) Add download binlog related configs to BE #47412

w41ter opened the pull request:
cherry pick from #47412

w41ter commented:
run buildall

Thearas commented:
Thank you for your contribution to Apache Doris.
Don't know what should be done next? See [How to process your PR](https://cwiki.apache.org/confluence/display/DORIS/How+to+process+your+PR).

Please clearly describe your PR:
1. What problem was fixed (it's best to include specific error reporting information). How it was fixed.
2. Which behaviors were modified. What was the previous behavior, what is it now, why was it modified, and what possible impacts might there be.
3. What features were added. Why was this function added?
4. Which code was refactored and why was this part of the code refactored?
5. Which functions were optimized and what is the difference before and after the optimization?

w41ter committed:
[chore](binlog) Add download binlog related configs to BE (#47412)

1. `download_binlog_meta_timeout_ms` the timeout of downloading or
getting meta
2. `enable_download_md5sum_check` whether to check the md5sum of the
download files, because it is a time-consuming operation

w41ter commented:
run buildall

w41ter commented:
run buildall
